_data/chains/eip155-14288640.json

```
@@ -0,0 +1,23 @@
+{
+  "name": "Anduschain Mainnet",
+  "chain": "anduschain",
+  "rpc": ["https://rpc.anduschain.io/rpc", "wss://rpc.anduschain.io/ws"],
+  "faucets": [],
+  "nativeCurrency": {
+    "name": "DAON",
+    "symbol": "DEB",
+    "decimals": 18
+  },
+  "infoURL": "https://anduschain.io/",
+  "shortName": "anduschain-mainnet",
+  "chainId": 14288640,
+  "networkId": 14288640,
+  "explorers": [
+    {
+      "name": "anduschain explorer",
+      "url": "http://explorer.anduschain.io",
+      "icon": "daon",
+      "standard": "EIP3091"
```

```suggestion
      "standard": "none"
```

the explorer does not seem to be EIP3091 compaible

---

I changed explorer standard and pushed.

It made new PR #2067

---

#2067 was my mistake. I closed

---

I changed explorer standard to none.
please check...